examples/deno.ts

```diff
@@ -0,0 +1,19 @@
+import createPlugin, { CurrentPlugin } from '../src/deno/mod.ts'
```

Comment:
Why do we include CurrentPlugin here?

Reply:
It probably had been imported for testing out host functions, will remove it